packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatTypeAssertions.ts

```diff
@@ -27,7 +27,10 @@ import { FileUploadAdapter } from '../../ChatComposite';
 
 /// CallWithChatAdapterManagement
 
-type CallWithChatAdapterManagementInternal = Omit<CallAdapterCallManagement, 'removeParticipant'> &
+type CallWithChatAdapterManagementInternal = Omit<
+  CallAdapterCallManagement,
+  'removeParticipant' | 'onReactionClicked'
```

Comment:
this shouldn't be needed here, CallWithChatComposite adapter should handle emojis the same as calladapter

Reply:
Pushing the change here in the subsequent PR with capability PR.